libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -1,66 +1,58 @@
 // (C) 2021 GoodData Corporation
-
 import * as Navigation from "../../tools/navigation";
 import { Table } from "../../tools/table";
 import { Widget } from "../../tools/widget";
 import { DrillToModal } from "../../tools/drillToModal";
-import { getBackend, getHost, getProjectId } from "../../support/constants";
-
+import { DateFilter } from "../../tools/dateFilter";
+import { DateFilterValue } from "../../tools/enum/DateFilterValue";
+import { Api } from "../../tools/api";
 //!!!!!!!!!!! this constant could be shifted check gray pages if test failed before each or after hook
 const DEPARTMENT_ID = "1090";
 //!!!!!!!!!!! this constant could be shifted check gray pages if test failed before each or after hook
```

Comment:
remove the comments `//!!!!!!!!!!! this constant could be shifted check gray pages if test failed before each or after hook`